packages/rke2-metrics-server/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://kubernetes-charts.storage.googleapis.com/metrics-server-2.11.1.tgz
```

Comment:
same. Make sure the chart.yaml is also renamed.

Reply:
this isnt renamed.